source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,41 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the `updateOne()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne>`_
+method. ``updateOne()`` takes a filter for a document, and updates the
+first document that matches the filter using an update document. The
+update document uses the :manual:`$set operator
```

Comment:
This statement isn't entirely [accurate](https://docs.mongodb.com/manual/reference/operator/update/#id1). I'd suggest rephrasing it to state that it accepts an available update operator to perform the desired update.

Reply:
Fixed:

* Changed the sentence to: "The update document uses the $set operator to set the value of a field in a document"

Reply:
That's what I mean though, `$set` is just one of many update operators available.